### PR TITLE
Standardize header styles for HHG+PPM SM flow

### PR DESCRIPTION
**Pull request by kahlouie**
## Description

Made some tweaks to standardize the header styles for the HHG + PPM Service member flow.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_populate_e2e
```

You can test the flow as the the hhgforppm@award.ed. Check that the spacing between the horizontal line and the title header are the same on each page, and the text for the wizard header is an h3 and the title header is an h2 on each page through the flow.

## Code Review Verification Steps

* [x] Design review approved
* [x] Request review from a member of a different team.
* [x] Have the Pivotal acceptance criteria been met for this change?

## References

* 

## Screenshots
New:
screen shot 2018-12-21 at 3 48 07 pm
Old:
screen shot 2018-12-21 at 3 59 13 pm

New:
screen shot 2018-12-21 at 3 48 13 pm
Old:
screen shot 2018-12-21 at 3 59 20 pm

New:
screen shot 2018-12-21 at 3 48 20 pm
Old:
screen shot 2018-12-21 at 3 59 36 pm

New:
screen shot 2018-12-21 at 3 48 27 pm
Old:
screen shot 2018-12-21 at 3 59 42 pm

New:
screen shot 2018-12-21 at 3 48 37 pm
Old:
screen shot 2018-12-21 at 3 59 50 pm

### CODE REVIEW

File: src/scenes/Legalese/index.css [hunk 1]
```diff
@@ -3,12 +3,19 @@
   justify-content: space-between;
 }
 
+h2 {
```
Comment (sarboc) [inline src/scenes/Legalese/index.css]: Is this going to be a standard for all of our headers, or just for these ones? If the latter, than keeping it here makes sense. If the former, then I think we need to look at a stylesheet with some larger reach.

File: src/scenes/Legalese/index.jsx [hunk 2]
```diff
@@ -80,48 +80,50 @@ export class SignedCertification extends Component {
...
-                <h2>Now for the official part...</h2>
-                <span className="box_top">
-                  <p className="instructions">
-                    Before officially booking your move, please carefully read and then sign the following.
-                  </p>
-                  <a className="pdf" onClick={this.print}>
-                    Print
-                  </a>
-                </span>
+              <div className="usa-width-one-whole">
```
Comment (sarboc) [inline src/scenes/Legalese/index.jsx]: Doesn't `usa-width-one-whole` only work when nested in a `grid` div?

Comment (kahlouie) [reply src/scenes/Legalese/index.jsx]: This component is already rendered inside a USA grid div. It's just getting called in the wizard page. This caused it to have margin-left and margin-right have some extra space on both sides of only the legalese page. So now it looks more in-line with the other pages.

File: src/scenes/Legalese/index.css (hunk 1 above)
Comment (kahlouie) [reply src/scenes/Legalese/index.css]: I think the former... I figured that would be part of the changes with the style guide, but I could be mistaken.

File: src/scenes/Legalese/index.jsx (hunk 2 above)
Comment (sarboc) [reply src/scenes/Legalese/index.jsx]: Gotcha. Thanks!

File: src/scenes/Legalese/index.css (hunk 1 above)
Comment (sarboc) [reply src/scenes/Legalese/index.css]: Yeah - I think we can clean that up as part of the style guide.

**Pull request merged**